static/js/robot.js

```diff
@@ -40,21 +40,32 @@ class RobotClass {
     wRadius = 2;                // radius of a wheel, inches
     MaxX = 144;                 // maximum X value, inches, field is 12'x12'
     MaxY = 144;                 // maximum Y value, inches, field is 12'x12'
-    neg = -1;                   // negate left motor calculation
-    startX = 70.0               // starting X position of the center of the robot
-    startY = 70.0               // starting Y position of the center of the robot
-
-    constructor() {
-      this.X = this.startX;     // current X position of the center of the robot
-      this.Y = this.startY;     // current Y position of the center of the robot
-      this.Wl = 0.0;            // angular velocity of l wheel, radians/s
-      this.Wr = 0.0;            // angular velocity of r wheel, radians/s
-      this.ltheta = 0.0;        // angular position of l wheel, degrees
-      this.rtheta = 0.0;        // angular position of r wheel, degrees
-      this.dir = 0.0;           // Direction of the robot facing, degrees
+    neg = -1;                    // negate left motor calculation
+    startX = 70.0
+    startY = 70.0
+
+    constructor(autonomous_main=null) {
```

Comment:
I don't think we're actually using this argument anymore, can this be confirmed?